client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -140,7 +140,7 @@ const Sidebar = ( { sidebarDomain, siteSlug, submit, goNext, goToStep, flow }: S
 						) }
 					</div>
 					{ sidebarDomain?.isWPCOMDomain && (
-						<a href={ `/domains/add/${ siteSlug }` }>
+						<a href={ `/domains/add/${ siteSlug }?is-launchpad-step=true` }>
```

Comment:
Does it make sense to use `import { addQueryArgs } from '@wordpress/url'` to add the query arg here? Or we also have `import { addQueryArgs } from 'calypso/lib/route'`.

Reply:
Yup `addQueryArgs` is a fine idea - I think that `calypso/lib/route` is just a wrapper around the WP one anyway